.github/workflows/testing.yml

```diff
@@ -54,5 +54,9 @@ jobs:
           python -m pip install --upgrade pip
             pip install .[strict,tests,docs]
 
-      - name: Test
-        run: pytest --cov=jobflow_remote --cov-report=xml
+      - name: Unit tests
+        run: pytest --cov=jobflow_remote --cov-report=xml --ignore tests/integration
+      
+      - name: Integration tests
+        # Unfortunate workaround required until we have better test/settings isolation
```

Comment:
> Yes, at the moment this will overwrite the coverage. Instead you can pass `--append` (or something) to get it to work, but as there are no other tests at the moment I think it's fine... Long-term I'm not sure how to fix this without more of a restructure, the problem is that the first jobflow_remote import loads the settings, which I need to overwrite to get the integration tests to work. So probably fine just to keep them separate forever and mark them not to run locally.
> 
> I'm still having some outstanding issues where the runner picks up the job but doesn't actually submit it in the queue (and does not make the queue database), hopefully will be able to iron these out soon...

I've seen this regarding grouping/separating coverage in codecov: https://docs.codecov.com/docs/flags
I think it would allow to say e.g. we want 100% coverage on unit tests and 95% on integration tests (or anything else, could even be 100 and 100 with specific hold out uncovered lines for each "type", unit/integration/..., of test).
Not urgent or absolutely necessary at this stage but could be useful.